common/lib/capa/capa/tests/test_targeted_feedback.py

```diff
@@ -96,8 +96,8 @@ def test_targeted_feedback_student_answer1(self):
 
         the_html = problem.get_html()
         without_new_lines = the_html.replace("\n", "")
-
-        self.assertRegexpMatches(without_new_lines, r"<targetedfeedback explanation-id=\"feedback3\">.*3rd WRONG solution")
+        # pylint: disable=line-too-long
+        self.assertRegexpMatches(without_new_lines, r"<targetedfeedback explanation-id=\"feedback3\" role=\"group\" aria-live=\"polite\" aria-describedby=\"1_2_1-legend\">.*3rd WRONG solution")
```

Comment:
Can you add coverage for the sr span that should appear immediatley before the targetedfeedback element?